### DISCUSSION TITLE
code-server stopped working, tried to restart, but does not work.

### DISCUSSION THREAD

**Discussion by miketaochina** [coder/code-server#2446] [Q&A]
Hi, I use a Mac, OSX 10.14.6 (18G87)

I installed code-server and it was work well, but suddenly it stopped working, so I entered `code-server` in terminal and it showed me this:
```
(base) MBP:~ me$ code-server
[2020-12-12T18:37:43.244Z] info  code-server 3.7.4 11f53784c58f68e7f4c5b3b8dae9407caa41725b
[2020-12-12T18:37:43.244Z] info  Using user-data-dir ~/.local/share/code-server
[2020-12-12T18:37:43.251Z] error listen EADDRNOTAVAIL: address not available `my public IP address here`:8080
```
it supposed to show me something end with `.yaml` which is `~/.config/code-server/config.yaml` but not any more, which I could not find my `password file`. But I searched Q&A and I found this command: `cat ~/.config/code-server/config.yaml` and I found it, which means this file is still there.
and then I tried to restart code-server, so I entered `sudo systemctl restart code-server`, and it showed me this:
```
(base) MBP:~ me$ sudo systemctl restart code-server
Password:
sudo: systemctl: command not found
```
Why this command not found..?
and I tried to scanned the ports which are open by `Nmap`, I could not find `port8080` in the list, which means it is not open.
so how i can make it work again?
Thanks a lot!

**Comment by kphanipavan** [comment]
```systemctl``` is only available on Linux OSs. Equivalent command for MacOS is ```launchctl```

Try 
```bash
sudo launchctl stop code-server
sudo launchctl start code-server
```

**Comment by kphanipavan** [comment]
Try 
![Screenshot 2020-12-13 at 9 56 55 AM](https://user-images.githubusercontent.com/60005847/102003146-8a68a780-3d29-11eb-86f6-331e07a6f64c.png)
to find ports which are being used...
